### PR TITLE
[0.18.4] Running yaml generator

### PR DESCRIPTION
**Pull request by matzew**
Signed-off-by: Matthias Wessendorf <mwessend@redhat.com>

### CODE REVIEW

File: openshift/release/knative-eventing-ci.yaml [hunk 1]
```diff
@@ -4184,6 +4184,7 @@ spec:
           image: registry.svc.ci.openshift.org/openshift/knative-v0.18.4:knative-eventing-mtping
           env:
             - name: SYSTEM_NAMESPACE
+              value: ''
```
Comment (slinkydeveloper) [inline openshift/release/knative-eventing-ci.yaml:4187]: ?!

Comment (slinkydeveloper) [reply openshift/release/knative-eventing-ci.yaml:4187]: value or value from?

Comment (aliok) [reply openshift/release/knative-eventing-ci.yaml:4187]: We need both: https://github.com/knative/eventing/pull/4472

**Comment by aliok** [comment]
/approve

Leaving lgtm to Mr. Slinky @slinkydeveloper

**Comment by slinkydeveloper** [comment]
/lgtm

Sorry i wasn't aware of the issue

**Pull request merged**